app/mailers/user_mailer.rb

```diff
@@ -317,15 +317,17 @@ def in_person_ready_to_verify(enrollment:, is_enhanced_ipp:)
     end
   end
 
-  def in_person_ready_to_verify_reminder(enrollment:)
+  def in_person_ready_to_verify_reminder(enrollment:, is_enhanced_ipp:)
     attachments.inline['barcode.png'] = BarcodeOutputter.new(
       code: enrollment.enrollment_code,
     ).image_data
 
+    @is_enhanced_ipp = is_enhanced_ipp
```

Comment:
Does this need to be a class variable on the `user_mailer`? I feel like `is_enhanced_ipp` should be accessed through the presenter.

Reply:
I tried to remove the instance variable `@is_enhanced_ipp`, but it broke specs in `user_mailer_spec.rb`.  I think we need this instance variable for the template to work.  I also think that's why @gina-yamada included this line 

>Set the value for  `@is_enhanced_ipp` in `in_person_ready_to_verify_reminder` in user_mailer

 in the ticket's Dev Notes.  (Gina, let me know if I'm missing anything.)

Reply:
Okay. I will resolve this comment.